packages/redux/src/raffles/types/actions.types.ts

```diff
@@ -0,0 +1,194 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type {
+  BlackoutError,
+  Raffle,
+  RaffleEstimation,
+  RaffleParticipation,
+} from '@farfetch/blackout-client';
+import type { NormalizedSchema } from 'normalizr';
+
+/**
+ * Payload of a raffle
+ */
+type RaffleSuccessPayload = NormalizedSchema<
+  {
+    raffles: Record<Raffle['id'], Raffle>;
+  },
+  Raffle['id']
+>;
+
+/**
+ * Actions dispatched when the raffle state request is made.
+ */
+export interface FetchRaffleRequestAction extends Action {
```

Comment:
These actions are not being used anywhere. They are supposed to be used inside each action factory for the dispatch calls.